### PR TITLE
Typo fix

### PR DESCRIPTION
**Pull request by ilia-cher**
Summary: fix a typo

Differential Revision: D15553921

### CODE REVIEW

**Review by dzhulgakov — APPROVED** [review]

File: aten/src/ATen/ParallelNative.cpp [hunk 1]
```diff
@@ -117,7 +117,7 @@ bool in_parallel_region() {
 
 void intraop_launch(std::function<void()> func) {
   if (!in_parallel_region()) {
-    internal::_get_intraop_pool.run([func](){
+    internal::_get_intraop_pool().run([func](){
```
Comment (yinghai) [inline aten/src/ATen/ParallelNative.cpp]: _get_intra_op_pool() might be more readable.

**Review by yinghai — APPROVED** [review]

**Comment by ezyang** [comment]
It doesn't seem like this typo caused any build failures. What happened?

**Comment by ilia-cher** [comment]
@ezyang you need to set AT_PARALLEL_NATIVE=1 to compile it, we should add a few CI builds with it

**Comment by ezyang** [comment]
@ilia-cher can you take point on adding the CI here? Alternately, if you can arrange to remove the compilation flag entirely, or minimize it so that only a small if statement is compiled optionally, that would be even better.

**Comment by ilia-cher** [comment]
I'll need some changes from @kostmo to add new CI builds, I don't there's a way to remove or minimize its usage, it's a switch between different parallel implementations

**Comment by ezyang** [comment]
My point is, even if you don't use one of the implementations, you could
still always compile both of them.

Excerpts from ilia-cher's message of 2019-05-31 12:44:20 -0700:
> @ezyang I'll need some changes from @kostmo to add new CI builds, I don't think there's a way to remove or minimize its usage, it's a switch between different parallel implementations
>

**Comment by ilia-cher** [comment]
@ezyang they're mutually exclusive, so without some trickery we can't link them both

**Pull request closed without merging**